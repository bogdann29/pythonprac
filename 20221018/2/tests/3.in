:print print()
:max x y e z max(x,y,e,z)
print
max 6 7 2 3
print
max 'a' 'b' 'c' 'd'
quit "{}, {}"