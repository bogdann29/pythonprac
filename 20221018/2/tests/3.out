
None
7

None
d
3, 7
